guide/english/ruby/common-array-methods/index.md

```diff
@@ -160,22 +162,20 @@ array.flatten
 ```
 
 #### .join
-The .join method returns a string of all the elements of the array separated by a separator parameter. If the separator parameter is nil, the method uses an empty string as a separator between strings.
+The .join method returns a string of all the elements of the array separated by a separator parameter. If the separator parameter is nil, the method uses an empty string as a separator between strings:
 
 ``` ruby
 array.join
-=> "1234"
+=> "01234"
 array.join("*")
-=> "1*2*3*4"
+=> "0*1*2*3*4"
 ```
 
 #### .each
-The .each method iterates over each element of the array, allowing you to perform actions on them.
+The .each method iterates over each element of the array, allowing you to perform actions on them:
 
 ``` ruby
-array.each do |element|
-  puts element
-end
+array.each { |element| puts element }
```

Comment:
same as the suggestion in the previous snippet, declare the array first for this snippet to work.